app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,41 @@
+class PostsController < ApplicationController
```

Comment:
Great job @haftamudesta 🤝

- [ ] As stated in the project requirements, kindly ensure for Post authorization, you use CanCan.

  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end